ssg/oval.py

```diff
@@ -18,11 +18,11 @@
 
 
 # globals, to make recursion easier in case we encounter extend_definition
-definitions = ET.Element("definitions")
-tests = ET.Element("tests")
-objects = ET.Element("objects")
-states = ET.Element("states")
-variables = ET.Element("variables")
+definitions = ET.Element("ns0:definitions")
```

Comment:
I think the more correct thing to do is register "oval" as a namespace and use it that way:

https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.register_namespace

Thoughts as to if this is preferred?

Reply:
Yes, I think that the current solution could crash if at some other place some other namespace got assigned the same prefix. I'm not an expert, but from what I read in the docs, using `register_namespace` should be correct.